src/XrdPosix/XrdPosixXrootd.cc

```diff
@@ -1060,6 +1060,22 @@ int XrdPosixXrootd::Readdir64_r(DIR *dirp, struct dirent64  *entry,
    return rc;
 }
 
+int XrdPosixXrootd::StatRet(DIR *dirp, struct stat *buf)
```

Comment:
Normally, methods are placed in this file are in alphabetic order so please place this after stafs().

Reply:
Done.